lib/faexport.rb

```diff
@@ -282,6 +283,27 @@ def set_content_type(type)
       end
     end
 
+    # /search/{query}/results.json
+    # /search/{query}/results.xml
+    # TODO: Implement RSS
+    get %r{/search/#{SEARCH_REGEX}/results\.(json|xml)} do |query, type|
```

Comment:
Since search queries can contain multiple words, spaces and possibly special characers, it might be better to put this in a query parameter.  Having a route of `%r{/search\.(json|xml)}` and using `params[:query]` would be cleaner IMO.